library/src/test/java/com/github/pwittchen/reactivenetwork/library/internet/observing/strategy/WalledGardenInternetObservingStrategyTest.java

```diff
@@ -51,11 +85,12 @@ private String getHost() {
 
   @Test public void shouldNotBeConnectedToTheInternet() {
     // given
-    when(strategy.isConnected(getHost(), PORT, TIMEOUT_IN_MS, errorHandler)).thenReturn(false);
+    String host = getHost();
+    Mockito.doReturn(false).when(strategy).isConnected(host, PORT, TIMEOUT_IN_MS, errorHandler);
```

Comment:
Please, use static import to exclude `Mockito` class name. It should look like: `doReturn(false).when(...)`. All tests in this project use that convention.